core/src/test-core/java/com/radixdlt/store/InMemoryTransactionsAndProofReader.java

```diff
@@ -124,21 +119,19 @@ public EventProcessor<LedgerUpdate> updateProcessor() {
   @Override
   public CommittedTransactionsWithProof getTransactions(DtoLedgerProof start) {
     synchronized (lock) {
-      final long stateVersion = start.getLedgerHeader().getAccumulatorState().getStateVersion();
+      final long startStateVersion =
+          start.getLedgerHeader().getAccumulatorState().getStateVersion();
       Entry<Long, CommittedTransactionsWithProof> entry =
-          store.committedTransactionRuns.higherEntry(stateVersion);
+          store.committedTransactionRuns.higherEntry(startStateVersion);
 
       if (entry != null) {
-        List<RawLedgerTransaction> transactions =
-            accumulatorVerifier
-                .verifyAndGetExtension(
-                    start.getLedgerHeader().getAccumulatorState(),
-                    entry.getValue().getTransactions(),
-                    t -> t.getLegacyPayloadHash().inner(),
-                    entry.getValue().getProof().getAccumulatorState())
-                .orElseThrow(RuntimeException::new);
-
-        return CommittedTransactionsWithProof.create(transactions, entry.getValue().getProof());
+        final var transactions = entry.getValue().getTransactions();
```

Comment:
I wonder if this might be clearer to be a method `getExtensionFrom(committedStateVersion)` on `CommittedTransactionsWithProof`? :)